Ntuplizer/test/ntuplize_cfg.py

```diff
@@ -193,16 +202,20 @@
 #############################################################################
 FlowSteps = []
 
-# everybody needs vertex cleaning
-from UWVV.AnalysisTools.templates.VertexCleaning import VertexCleaning
-FlowSteps.append(VertexCleaning)
+# Vertex cleaning for ZZ
+if zz:
```

Comment:
Ok, I'll turn it off just for WZ. I spent some time looking into this and understood that the general recommendation is not to include this filter for current data. A very similar condition is, however, applied in the MET filters, though in this case the events would be rejected for having a bad primary vertex rather than allowing another vertex to be promoted.